Makefile

```diff
@@ -65,7 +65,6 @@ clean:
 		$(UI_BUILD_DIR) \
 		pkg/actions/mock \
 		pkg/api/lakefs.gen.go \
-		pkg/ddl/statik.go \
 		pkg/graveler/sstable/mock \
 		pkg/webui \
```

Comment:
I *think* this should also be removed... but I am not sure.  Could you either explain what goes in each `webui` directory, or remove this line and `fgrep -R webui` to be sure we got all bits?

Reply:
Right, I missed this one - thanks!

Reply:
Removing the extra directory - it was left for the cleanup and it is no longer used.
We will keep content.go file under the real webui to include the UI dist folder.